pkgs/development/python-modules/pyspinel/default.nix

```diff
@@ -1,15 +1,23 @@
-{ buildPythonPackage, fetchPypi, lib, future, pyserial, ipaddress }:
+{ lib, buildPythonPackage, fetchFromGitHub
+, future, pyserial, ipaddress
+}:
 
 buildPythonPackage rec {
   pname = "pyspinel";
-  version = "1.0.1";
+  version = "unstable-2020-06-19";  # no versioned release since 2018
```

Comment:
```suggestion
  version = "unstable-2020-06-19";  # no versioned release since 2018
  disabled = isPy27;
```
```
builder for '/nix/store/22xb05zhil1mgzimjhpwb4f7xvlbmvmj-python2.7-pyspinel-unstable-2020-06-19.drv' failed with exit code 1; last 10 log lines:
  no configure script, doing nothing
  building
  Executing setuptoolsBuildPhase
  Traceback (most recent call last):
    File "nix_run_setup", line 8, in <module>
      exec(compile(getattr(tokenize, 'open', open)(__file__).read().replace('\\r\\n', '\\n'), __file__, 'exec'))
    File "setup.py", line 41
      file=sys.stderr)
          ^
  SyntaxError: invalid syntax
cannot build derivation '/nix/store/0n4grwwwz1c006z03mfsd8j5ja1hzkvw-env.drv': 1 dependencies couldn't be built
```

Reply:
Apologies, will be more diligent checking and annotating the Py2 compatibility of packages